src/resources/js/event-editor.js

```diff
@@ -44,7 +44,7 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 	 *
 	 * @since 6.0.1
 	 */
-	obj.init = () => {
+	obj.init = function(){
```

Comment:
Lets keep the other one.

Reply:
@bordoni I disagree. All other functions in this file are not arrow functions, plus we would have to explicitly ignore the `Unexpected arrow-function expression` error that `eslint` raises.

Reply:
@bordoni Do you mind taking a look at this again? I'd like to get it merged into the next MR if possible and make this test required.